website/content/en/status/report-2023-07-2023-09/mfsbsd-build-integration.adoc

```diff
@@ -0,0 +1,30 @@
+=== Integrate mfsBSD into the Release Building Tools
+
+Links: +
+link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[Wiki Article] URL: link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[]+
+link:https://reviews.freebsd.org/D41705[Code Review on Phabricator] URL: link:https://reviews.freebsd.org/D41705[]+
+link:https://freebsdfoundation.org/blog/meet-the-2023-freebsd-google-summer-of-code-students-soobin-rho/[FreeBSD Foundation Blog Post] URL: link:https://freebsdfoundation.org/blog/meet-the-2023-freebsd-google-summer-of-code-students-soobin-rho/[]
+
+Contact: Soobin Rho <soobinrho@FreeBSD.org>
+
+==== What is mfsBSD?
+
+"mfsBSD is a toolset to create small-sized but full-featured mfsroot based distributions of FreeBSD that store all files in memory (MFS) [Memory File System] and load from hard drive, USB storage device, or optical medium.
```

Comment:
"Optical medium" seems wrong here. It is pedanticly correct, but obscure. 
I'd be tempted to use "cdrom" instead. It's more approachable and accurate enough to convey the meaning without delving down into the obscure.

As a side note: why not a unified image for both iso and img? Most other projects have long since gone that way and we have at least some support for that in our tree.

Reply:
Thank you, @bsdimp!

- Good point! However, I'd be wary of changing the wording here because it's a quote from the [mfsBSD white paper](https://people.freebsd.org/~mm/mfsbsd/mfsbsd.pdf).
- This mfsBSD integration project just imports mfsBSD into the FreeBSD release tools, and I don't actually have control over the upstream mfsBSD source code. Nice idea nonetheless. Thanks guys 👍

Reply:
Ok on the quote.
There's an issue that I have raised in the code review though that might require changes to mfsbsd...

Reply:
Hey @bsdimp, I'll definitely research about the unified image format in mfsBSD. Thank you for pointing me to the right direction. It will be very helpful! Making changes to the code revision now.

Reply:
You bet. I also understand, btw, GSoC timelines and scopes of project. I'm not looking to make things difficult for you if my questions/comments lead to something unreasonable... but I'm happy to help make things better.